docs/content/documentation/systems/Materials/phase_field/IdealGasFreeEnergy.md

```diff
@@ -0,0 +1,17 @@
+# IdealGasFreeEnergy
+!syntax description /Materials/IdealGasFreeEnergy
+
+This material defines the Helmholtz free energy density $f$ of an ideal Boltzmann gas.
+
+$$
+F = U - TS = -Nk_BT\left(1+\ln\frac {n_Q}n\right)
```

Comment:
I would put f = F/V on LHS instead of F, then you can put in n = N/V on the RHS so you have everything in terms of n